app/src/main/res/values/strings.xml

```diff
@@ -118,7 +118,7 @@
     <string name="recent_tabs_show_all">Show all</string>
 
     <!-- History Metadata -->
-    <!-- Header text for past explorations in the home screen -->
+    <!-- Header text for grouped highlights from the user's browsing history e.g., topics they have researched (explored) on the web in the home screen -->
```

Comment:
nit: Would rephrase now that I see the whole thing :). Header text for a section on the home screen that displays grouped highlights from the user's browsing history, such as topics they have researched or explored on the web.